pkg/reloader/reloader.go

```diff
@@ -363,3 +392,17 @@ func expandEnv(b []byte) (r []byte, err error) {
 	})
 	return r, err
 }
+
+func getFileInode(filepath string) (uint64, error) {
+	fileinfo, err := os.Stat(filepath)
+	if err != nil {
+		return 0, err
+	}
+
+	stat, ok := fileinfo.Sys().(*syscall.Stat_t)
+	if !ok {
+		return 0, errors.New("Not a syscall.Stat_t")
```

Comment:
Typically errors start with a lower-case letter so that it would look nice if they will be chained